src/patternfly/components/Form/form.scss

```diff
@@ -59,7 +58,6 @@
 
     @media (min-width: $pf-global--breakpoint--md) {
       .pf-c-form__group {
-        // this number needs to be confirmed by design :)
         grid-template-columns: 150px 1fr;
```

Comment:
Not from you but I guess this should be variable right?

Reply:
Good call, seems like something someone might want to change.